akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -384,10 +388,6 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
   // !!! WARNING !!! This is *NOT* thread safe,
   private val topLevelFREvents = flightRecorder.createEventSink()
 
-  // FIXME: Compression table must be owned by each channel instead
-  // of having a global one
-  val compression = new Compression(system)
```

Comment:
is now hosted in association state.
While it is true that we may be getting compression requests before the association is ready, this is OK, since then we say "outgoing compression not ready yet" (you'll see that in the encoder)